### DISCUSSION TITLE
Is there way to replace default startup error page (i.g 500.30) in web app hosted on azure

### DISCUSSION THREAD

**Discussion by Kroshtandt** [dotnet/aspnetcore#24872] [General]
I read this [article](https://docs.microsoft.com/en-us/aspnet/core/test/troubleshoot-azure-iis?view=aspnetcore-2.2), but seems that I can only turn on/off ASP.NET Core Module stdout log. 
I use .NET Core 2.2

ASPNETCORE_ENVIRONMENT is Production, and I don't want to show startup error page to my users, just to show some static content (html).

![image](https://user-images.githubusercontent.com/14110323/90140406-63ea2e00-dd82-11ea-8d04-cdbe552bcf46.png)

**Comment by Tratcher** [comment]
No, that's an error page of last resort. If it couldn't load your site there's no guarantee it would be able to load a custom error page.

These errors should only happen after deployment, so the impact to your users should be limited.

Were you able to track down why it didn't start?

**Reply by Kroshtandt** [reply]
Hello, thank you for your reply!
The issue was because database log table overflow. 
What do you think - is this possible to handle configure that issue on azure or IIS level? 

Br

**Reply by Tratcher** [reply]
> What do you think - is this possible to handle configure that issue on azure or IIS level?

Configure the error page, or configure the log table?

**Reply by Kroshtandt** [reply]
I mean configure this error page on azure or IIS level
